src/Pages/LogoutPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace WTSA1\Pages;
+use WTSA1\Engines\Session;
+
+class LogoutPage extends Page {
+    protected function post() {
```

Comment:
This uses post so no funny people send logout links to their friends :)

Reply:
Cross-Site-Request-Forgery

Reply:
Valid point. What do you think about leaving this as is and adding an optimization task (M7) to add CSRF token generation to our framework so that they can be used for all forms?

Reply:
> Valid point. What do you think about leaving this as is and adding an optimization task (M7) to add CSRF token generation to our framework so that they can be used for all forms?

Yea lets add it as an optional issue for M7 👍 , may you create it?